### PR TITLE
Rework multi acquire

### PR DESCRIPTION
**Pull request by Brow71189**
I think you had some comments on the reworked code which is why we did not merge the changes yet. I'm just opening this so that we don't forget about it.

### CODE REVIEW

**Comment by cmeyer** [comment]
Is this ready for review or still a work in progress? (We can mark it as a 'draft' if it's still in progress in the Reviewers section at the top-right of this web page).

**Comment by Brow71189** [comment]
It is ready by itself an contains hooks for implementing drift correction.
Just actually implementing drift correction will need more planning and
diacussion.

Chris Meyer <notifications@github.com> schrieb am Mi., 11. Nov. 2020, 17:05:

File: nion/instrumentation/MultiAcquire.py [hunk 1]
```diff
@@ -70,96 +82,329 @@ def copy(self):
         return self.__copy__()
 
 
+class ScanDataChannel(scan_base.SynchronizedDataChannelInterface):
```
Comment (cmeyer) [inline nion/instrumentation/MultiAcquire.py:85]: I don't think this should be derived from scan_base.SynchronizedDataChannelInterface, which is for cameras. The 'update' method conflicts in my type checker.

**Review by cmeyer — CHANGES_REQUESTED** [review]
Overall this PR looks good - much cleaner code than before.

There is one outstanding issue - I think ScanDataChannel should be a standalone class, not derived from SynchronizedDataChannelInterface. The 'update' method conflicts in the type checker.

Is it ok to squash these changes into a single commit? Or rebase to consolidate the commits into 1-3 commits? If you don't have any opinion, I'll just squash before merging.

Also, no changes required for this PR, but one thing that would be useful in the future is to run it through mypy and flake8 for type checking and formatting.

**Comment by cmeyer** [comment]
Merged 3500de95717663ce341b4534d184b56b5df2133d

**Pull request closed without merging**